### ISSUE TITLE
What is the traing v.s. validation loss on your trained model

### ISSUE THREAD

**Issue by riven314** [sgrvinod/a-PyTorch-Tutorial-to-Image-Captioning#121]
Thanks for the tutorial and code implementation!

I am trying to train the model on my custom dataset and I encountered high loss on both training dataset and validation dataset (~5.8-7.0 loss). But oddly the accuracy is fair (40-60%) given the high loss. 

May I ask you what is the loss value you got by the end of your trained model. It could be a reference to me whether my loss value makes sense/ could be a metrics to the performance.

**Comment by enes3774** [comment]
hi, I had nearly the same loss(5.). Do you think the model learns or not? My accuracy is about %50
